darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1281,11 +1280,10 @@ def predict(
         num_samples
             Number of times a prediction is sampled from a probabilistic model. Should be left set to 1
             for deterministic models.
-        num_loader_workers
-            Optionally, an integer specifying the ``num_workers`` to use in PyTorch ``DataLoader`` instances,
-            for the inference/prediction dataset loaders (if any).
-            A larger number of workers can sometimes increase performance, but can also incur extra overheads
-            and increase memory usage, as more batches are loaded in parallel.
+        data_loader_kwargs
+            Optionally, a dictionary of keyword arguments to pass to the PyTorch DataLoader instances used to load the
```

Comment:
it's referring to the train and val datasets, but it should be for the prediction dataset, same for the methods below